tests/Symbolica.Extensions.Configuration.FSharp.Tests/BinderTests.fs

```diff
@@ -15,6 +13,16 @@ module Result =
     let ``should create a Binder that ignoes the config and returns Success`` (x: int) (config: string) =
         test <@ x |> Binder.result |> Binder.eval config = Success(x) @>
 
+module Fail =
+    [<Property>]
+    let ``should create a Binder that ignoes the config and returns Failure`` (e: string) (config: string) =
```

Comment:
You have an "ignoes" here and also just above. It's not part of this PR but you also have an "evaulates" in `Binder.fs`.

Reply:
Yeah I spotted the evaluates one and fixed in a later PR. I'll do ignoes now though.

Reply:
Will fix these in the next PR.